cluster/gce/gci/mounter/stage-upload.sh

```diff
@@ -51,8 +51,8 @@ function cleanup {
 # Delete temporary directories on exit
 trap cleanup EXIT
 
-mkdir ${RKT_DIR}
-mkdir ${ACI_DIR}
+mkdir -p ${RKT_DIR}
+mkdir -p ${ACI_DIR}
```

Comment:
This isn't necessary. Why should we add it?

Reply:
sorry,I have update it!